src/modules/users/components/TokenActivation/TokenActivationPopover.tsx

```diff
@@ -4,9 +4,8 @@ import Popover, { PopoverChildFn } from '~core/Popover';
 
 import { removeValueUnits } from '~utils/css';
 
-import TokenActivationContent, {
-  TokensTabProps,
-} from './TokenActivationContent/index';
+import TokenActivationContent from './TokenActivationContent/index';
+import { TokensTabProps } from './TokensTab/TokensTab';
```

Comment:
You can use shorter imports here:
```
import TokenActivationContent from './TokenActivationContent';
import { TokensTabProps } from './TokensTab';
```

For the `TokensTabProps`, you would need to add the export to the index file of the `TokensTab`:
```
export { default, TokensTabProps } from './TokensTab';
```